pubtools/_quay/tag_images.py

```diff
@@ -277,7 +285,7 @@ def tag_images(
         dest_host = dest_refs[0].split("/", 1)[0]
         source_host = source_ref.split("/", 1)[0]
         executor.skopeo_login(dest_host, quay_user, quay_password)
-        if source_quay_user and source_quay_password and dest_host != source_host:
+        if source_quay_host and source_quay_user and source_quay_password:
             executor.skopeo_login(source_host, source_quay_user, source_quay_password)
```

Comment:
source_quay_host is not always set.

Reply:
If source_quay_host is not set, it will not execute line 289 "executor.skopeo_login(source_host, source_quay_user, source_quay_password)",  the variable `source_host` will be not used. 

If we change line 289 to "executor.skopeo_login(source_quay_host, source_quay_user, source_quay_password)", I don't see any impact for the logic without line 286.

Reply:
You are right, updated :)